src/main/scala/mesosphere/marathon/api/v2/LabelSelector.scala

```diff
@@ -1,16 +1,17 @@
 package mesosphere.marathon.api.v2
 
+import mesosphere.marathon.core.appinfo.AppSelector
 import mesosphere.marathon.state.AppDefinition
 import org.apache.log4j.Logger
 
 import scala.util.control.NonFatal
 import scala.util.parsing.combinator.RegexParsers
 
-case class LabelSelector(key: String, fn: String => Boolean, value: List[String]) {
+case class LabelSelector(key: String, fn: String => Boolean, value: List[String]) extends AppSelector {
```

Comment:
Since the LabelSelector now is an AppSelector - should the class be moved to the core.appinfo(.impl)